packages/services/src/contents/index.ts

```diff
@@ -637,8 +642,40 @@ export class ContentsManager implements Contents.IManager {
   constructor(options: ContentsManager.IOptions = {}) {
     const serverSettings = (this.serverSettings =
       options.serverSettings ?? ServerConnection.makeSettings());
-    this._defaultDrive = options.defaultDrive ?? new Drive({ serverSettings });
-    this._defaultDrive.fileChanged.connect(this._onFileChanged, this);
+    // Create a drive for the Jupyter Server REST API.
+    let contentsDrive = new Drive({ serverSettings, name: 'contents' });
```

Comment:
The default drive name is `Default`

https://github.com/jupyterlab/jupyterlab/blob/main/packages/services/src/contents/index.ts#L1079

So although it is not a good name if this PR is merged, I would not change it for backward compatibility.

Reply:
:+1: